packages/remote-feature-flag-controller/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@metamask/remote-feature-flag-controller",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
The package will not get published until we make a new release that includes it, so there is no worry about accidentally publishing it yet. (The version is also set to `0.0.0` so that is serving as a prevention measure already in the meantime.)